.classpath

```diff
@@ -24,5 +24,7 @@
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/gcs dependencies"/>
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/google-api-client-appengine-1.18.0-rc.jar"/>
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/google-api-client-1.18.0-rc.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/repackaged-appengine-jakarta-jstl-1.1.2.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/repackaged-appengine-jakarta-standard-1.1.2.jar"/>
```

Comment:
Is this the latest version? If not, are we going to upgrade to the latest version?